index.html

```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
+            <div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-3.png" alt="project-snapshot">
+            <h2 class="project-title">Tonic</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
+            <div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-4.png" alt="project-snapshot">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
+            <div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+    </section>
+    <section class="about-me">
+        <div class="about-container">
+            <h1 class="about-title">About Myself</h1>
+            <div class="about-body">
+                <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
```

Comment:
- [ ] You should stick to the design as much as possible. pleas adjust the spacing, see image below for reference
### Your design
![your](https://user-images.githubusercontent.com/81727988/212335722-38d9c955-0ab4-41ed-aca1-0447d874f63a.jpg)


### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/212335749-a3edebce-affd-4492-bad6-ecb1bfb87a10.jpg)

Reply:
![about-self](https://user-images.githubusercontent.com/47403388/212368528-3c732f68-138c-47e3-892b-e34f1678d49d.jpg)